pallets/parachain-staking/src/mock.rs

```diff
@@ -136,6 +161,7 @@ parameter_types! {
 	pub const MinCollatorStk: u128 = 10;
 	pub const MinDelegatorStk: u128 = 5;
 	pub const MinDelegation: u128 = 3;
+	pub const RewardPotId: PalletId = PalletId(*b"av/vamgr");
```

Comment:
Shall we modify this identifier to conform to the current pallet?

Reply:
Keeping this the same so we generate the same account address